Dockerfile

```diff
@@ -67,10 +73,10 @@ RUN swupd update ${SWUPD_UPDATE_ARG} && swupd bundle-add file xfsprogs storage-u
 RUN ldconfig
 
 # move required binaries and libraries to clean container
-COPY --from=build /usr/lib64/libndctl.so.* /usr/lib/
-COPY --from=build /usr/lib64/libdaxctl.so.* /usr/lib/
+COPY --from=binaries /usr/lib64/libndctl.so.* /usr/lib/
```

Comment:
As we build `ndctl` in the build phase, `--from=build `is appropriate right?

Reply:
Both works. I find copying from  "binaries" better because then we get everything from a single source image. Not sure whether that has any impact, though,